cellprofiler/modules/imagemath.py

```diff
@@ -171,28 +185,28 @@ def add_image(self, removable=True):
         group = cellprofiler.setting.SettingsGroup()
         group.removable = removable
         group.append("image_or_measurement", cellprofiler.setting.Choice(
-                "Image or measurement?", [IM_IMAGE, IM_MEASUREMENT], doc="""
-            You can perform math operations using two images or you
-            can use a measurement for one of the operands. For instance,
-            to divide the intensity of one image by another, choose <i>%(IM_IMAGE)s</i>
-            for both and pick the respective images. To divide the intensity
-            of an image by its median intensity, use <b>MeasureImageIntensity</b>
-            prior to this module to calculate the median intensity, then
-            select <i>%(IM_MEASUREMENT)s</i> and use the median intensity measurement as
-            the denominator""" % globals()))
-
-        group.append("image_name", cellprofiler.setting.ImageNameSubscriber("", "", doc="""
-            Selec the image that you want to use for this operation."""))
+                "Image or measurement?", [IM_IMAGE, IM_MEASUREMENT], doc="""\
+You can perform math operations using two images or you can use a
+measurement for one of the operands. For instance, to divide the
+intensity of one image by another, choose *%(IM_IMAGE)s* for both and
+pick the respective images. To divide the intensity of an image by its
+median intensity, use **MeasureImageIntensity** prior to this module to
+calculate the median intensity, then select *%(IM_MEASUREMENT)s* and
+use the median intensity measurement as the denominator
+""" % globals()))
+
+        group.append("image_name", cellprofiler.setting.ImageNameSubscriber("", "", doc="""\
+Select the image that you want to use for this operation."""))
 
         group.append("measurement", cellprofiler.setting.Measurement(
-                "Measurement", lambda: cellprofiler.measurement.IMAGE, "", doc="""
-            This is a measurement made on the image. The value of the
-            measurement is used for the operand for all of the pixels of the
-            other operand's image."""))
-
-        group.append("factor", cellprofiler.setting.Float("", 1, doc="""
-            Enter the number that you would like to multiply the above image by. This multiplication
-            is applied before other operations."""))
+                "Measurement", lambda: cellprofiler.measurement.IMAGE, "", doc="""\
+This is a measurement made on the image. The value of the
+measurement is used for the operand for all of the pixels of the
+other operand's image."""))
+
+        group.append("factor", cellprofiler.setting.Float("", 1, doc="""\
```

Comment:
Please change to group.append("factor", cellprofiler.setting.Float("Multiply the image by", 1, doc="""\